source/index.html.md

```diff
@@ -474,6 +474,9 @@ The following objects can be used as parameters in the *include* query
 | files/packingList                       | Details of the group's Packing Lists                               |
 | files/packingList/lineItem              | List of line items associated with the Packing List                |
 | files/apInvoice                         | Details of the Accounts Payable Invoice                            |
+| files/apInvoice/cluster                 | List of clusters associated with Payable Invoice                   |
+| files/apInvoice/cluster/jobReference    | List of References associated with Payable Invoice's cluster       |
+| files/apInvoice/cluster/extractedLine   | List of extracterd Line associated with Payable Invoice'cluster    |
```

Comment:
There is a spelling mistake in Invoice's cluster ..
And also may be it is better to say "List of extracted charge lines associated with Payable invoice's cluster"